robot/rover/Arm/src/commands/ArmCommandCenter.cpp

```diff
@@ -1,131 +1,77 @@
+        float* angles = (float*)malloc(sizeof(*angles) * numAngles);
 
-    switch(commandID)
-    {
-        case COMMAND_EMERGENCY_STOP: {
-            emergencyStop();
-            break;
-                                     }
-        case COMMAND_REBOOT_TEENSY: {
-            rebootTeensy();
-            break;
-                                    }
-        case COMMAND_STOP_MOTORS: {
-            stopAllMotors();
-            break;
-                                  }
-        case COMMAND_RESET_ANGLES: {
-            resetAngles();
-            break;
-                                   }
-        case COMMAND_HOME_MOTORS: {
-            homeAllMotors(*rawArgs);
-            break;
-                                  }
-        case COMMAND_HOME: {
-            homeMotor(*rawArgs, *(++rawArgs));
-            break;
-                           }
-        case COMMAND_ARM_SPEED: {
-            setArmSpeed(bytes_to_float(rawArgs));
-            break;
-                                }
-        case COMMAND_STOP_SINGLE_MOTOR:{
-            stopSingleMotor(*rawArgs);
-            break;
-                                       }
-        case COMMAND_GEAR_RATIO: {
-            setGearRatioValue(*(rawArgs++), bytes_to_float(rawArgs));
-            break;
-                                 }
-        case COMMAND_OPEN_LOOP_GAIN: {
-            setOpenLoopGain(*(rawArgs++), bytes_to_float(rawArgs));
-            break;
-                                     }
-        case COMMAND_PID_CONSTANTS: {
-            setPidConstants(*(rawArgs++), bytes_to_float(rawArgs), bytes_to_float(rawArgs), bytes_to_float(rawArgs));
-            break;
-                                    }
-        case COMMAND_OPEN_LOOP_STATE: {
-            setOpenLoopState(*(rawArgs++), *(rawArgs) == 1);
-            break;
-                                      }
-        case COMMAND_RESET_SINGLE_MOTOR: {
-            resetSingleMotor(*rawArgs);
-            break;
-                                         }
-        case COMMAND_BUDGE_MOTORS: {
-            budgeMotors(rawArgs);
-            break;
-                                   }
-        case COMMAND_MOVE_MULTIPLE_MOTORS: {
-            moveMultipleMotors((byte*)rawArgs);
-            break;
-                                           }
-        case COMMAND_PING: {
-            pong();
-            break;
-                           }
-        case COMMAND_GET_MOTOR_ANGLES: {
-            printMotorAngles();
-            break;
-                                      }
+        for (int i = 0; i < numAngles; i++) {
+          angles[i] = bytes_to_float(rawArgs + i * sizeof(float));
+        }
+
+        setMotorSpeeds(angles);
+        free(angles);
+        break;
+      } else {
+        invalidCommand(cmdID, rawArgs, rawArgsLength);
+        break;
+      }
+    }
+    case COMMAND_DEBUG_TEST:
+      debug_test();
+      break;
+    default: {
+      invalidCommand(cmdID, rawArgs, rawArgsLength);
```

Comment:
This will trigger for all wheel commands. Maybe it can be wrapped in `#ifdef DEBUG`?

Reply:
How is this undesirable?